modules/veteran_verification/VETERAN_VERIFICATION.yml

```diff
@@ -88,6 +88,8 @@ paths:
                 properties:
                   data:
                     $ref: '#/components/schemas/VeteranStatusConfirmation'
+        '401':
+          description: Unauthorized.
```

Comment:
For consistency with the returned error, recommend updating to `Not authorized` for the 401 descriptions.